docs/java-rest/high-level/textstructure/find-structure.asciidoc

```diff
@@ -1,30 +1,30 @@
 --
-:api: find-file-structure
-:request: FindFileStructureRequest
-:response: FindFileStructureResponse
+:api: find-structure
+:request: FindStructureRequest
+:response: FindStructureResponse
 --
 [role="xpack"]
 [id="{upid}-{api}"]
-=== Find file structure API
+=== Find structure API
 
 experimental::[]
 
-Determines the structure of a text file and other information that will be
+Determines the structure of text and other information that will be
 useful to import its contents to an {es} index. It accepts a +{request}+ object
 and responds with a +{response}+ object.
 
 [id="{upid}-{api}-request"]
-==== Find file structure request
+==== Find structure request
 
-A sample from the beginning of the file (or the entire file contents if
+A sample from the beginning of the text (or the entire text contents if
```

Comment:
This makes sentence makes less sense once file is replaced with text especially the entire contents bit. 

```suggestion
A sample of the the text to analyze must be added to the +{request}+ object using the....
```